### DISCUSSION TITLE
what kind of fonts are you using?

### DISCUSSION THREAD

**Discussion by zhuzhzh** [askfiy/nvim#31] [General]
![image](https://user-images.githubusercontent.com/14127098/234546127-84e0ac30-7943-4b7d-8a5f-86cf24ea9583.png)

我使用的是Hack Nerd Font Mono Regular, 界面上有些乱码

**Comment by zhuzhzh** [comment]
the icons of vscode is not shown.

**Comment by askfiy** [comment]
You need to patch fonts to display icons

https://www.nerdfonts.com/
